src/Message/VirusScan.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Message;
+
+/**
+ * Message for virus-scanning files.
+ */
+class VirusScan
+{
+    /**
+     * The file ID of the file to be scanned.
+     *
+     * @var string
+     */
+    protected $fileId;
+
+    /**
+     * Constructor.
+     *
+     * @param int $fileId The file ID of the file to be scanned.
+     */
+    public function __construct(int $fileId)
+    {
+        $this->fileId = $fileId;
```

Comment:
I think we should change this to filepath as a constructor, this way it can be independent if a file entity exists.